app/code/Magento/ConfigurationImportMessageBroker/Model/Configuration/Import/Mapping/SchemaLocator.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\ConfigurationImportMessageBroker\Model\Configuration\Import\Mapping;
+
+use Magento\Framework\Module\Dir;
+
+class SchemaLocator implements \Magento\Framework\Config\SchemaLocatorInterface
+{
+    const CONFIG_FILE_SCHEMA = 'configuration_import_mapping.xsd';
+
+    /**
+     * Path to corresponding XSD file with validation rules for merged config
+     *
+     * @var string
+     */
+    protected $schema;
```

Comment:
please, make it private here and in other places where it's possible

Reply:
replaced